test/e2e/nns_update_timestamp_test.go

```diff
@@ -0,0 +1,47 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var _ = Describe("NNS update timestamp", func() {
+	Context("when updating nns", func() {
+		It("timestamp should be changed", func() {
+			for _, node := range nodes {
+				var interval int = 15
+				key := types.NamespacedName{Name: node}
+				timeBefore := nodeNetworkState(key).Status.LastSuccessfulUpdate
+
+				if kubeconfig := os.Getenv("KUBECONFIG"); len(kubeconfig) > 0 {
+					config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+					ExpectWithOffset(1, err).ToNot(HaveOccurred())
+
+					clientset, err := kubernetes.NewForConfig(config)
+					cs, err := clientset.CoreV1().ConfigMaps("nmstate").Get("nmstate-config", metav1.GetOptions{})
+					ExpectWithOffset(1, err).ToNot(HaveOccurred())
```

Comment:
You can simplify this using operator-sdk testing framework to get the client

https://github.com/operator-framework/operator-sdk/blob/7d3979536504bdd74b08456313c34e74d947d514/test/e2e/tls_util_test.go#L98

```
	configMap, err := framework.Global.KubeClient.CoreV1().ConfigMaps("nmstate").Get("nmstate-config", metav1.GetOptions{})
```